catalog.html

```diff
@@ -1,15 +1,383 @@
+                    </article>
+                </li>
+                <li class="list-item">
+                    <article class="list-product">
+                        <img src="#" alt="photo" class="product-image">
+                        <h3 class="product-title">Artique</h3>
+                        <p class="product-sub">Creative Agency Website Design</p>
+                        <p class="product-price">$12.00</p>
+                        <button type="button" class="cart-button">
+                            <img src="#" alt="cart-logo"></button>
+                    </article>
+                </li>
+                <li class="list-item">
+                    <article class="list-product">
+                        <img src="#" alt="photo" class="product-image">
+                        <h3 class="product-title">MindTrix</h3>
+                        <p class="product-sub">AI Tools Website Design</p>
+                        <p class="product-price">$5.00</p>
+                        <button type="button" class="cart-button">
+                            <img src="#" alt="cart-logo"></button>
+                    </article>
+                </li>
+                <li class="list-item">
+                    <article class="list-product">
+                        <img src="#" alt="photo" class="product-image">
+                        <h3 class="product-title">Koki</h3>
+                        <p class="product-sub">Restaurant Admin Dashboard</p>
+                        <p class="product-price">$20.00</p>
+                        <button type="button" class="cart-button">
+                            <img src="#" alt="cart-logo"></button>
+                    </article>
+                </li>
+            </ul>
+            <div class="pagination">
+                <a href="#" class="previous-page">&#8249;</a>
+                <a href="#" class="page-number">01</a>
+                <a href="#" class="page-number">02</a>
+                <a href="#" class="page-number">03</a>
+                <a href="#" class="page-more">...</a>
+                <a href="#" class="page-number">10</a>
+                <a href="#" class="next-page">&#8250;</a>
+            </div>
+            <div class="cta-section">
+                <img src="#" alt="#" class="background-figure">
+                <img src="#" alt="#" class="background-figure">
+                <img src="#" alt="#" class="background-figure">
+                <img src="#" alt="#" class="background-figure">
+                <img src="#" alt="#" class="background-figure">
+                <img src="#" alt="#" class="background-figure">
+                <img src="#" alt="#" class="background-figure">
+                <img src="#" alt="#" class="background-figure">
+                <img src="#" alt="#" class="background-figure">
+                <img src="#" alt="#" class="background-figure">
+                <label id="cta-label">PRODUCT</label>
```

Comment:
we can use `label `tag **only** inside `form` tag
because it used for styling inputs